### PR TITLE
Specified versions for each dependency

### PR DESCRIPTION
**Pull request by jammy270**
Modified setup.py to specify versions for each dependency

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -20,18 +20,18 @@
 	install_requires=[
 	'scipy==1.5.4',
 	'matplotlib==3.3.3',
-	'tensorflow~=2.3.3',
-	'tensorflow-addons',
-	'dill',
+	'tensorflow==2.3.4',
```
Comment (ashishsingh18) [inline setup.py:23]: Can we keep the tensorflow version same as before?

**Review by ashishsingh18 — APPROVED** [review]
minor change requested.

File: setup.py (hunk 1 above)
Comment (jammy270) [reply setup.py]: Sure, its just that when I run pip freeze on the Alpha2 version container, I see 2.3.4 was picked. I have now changed it back to 2.3.3

**Review by ashishsingh18 — APPROVED** [review]
looks fine

**Pull request merged**